### PR TITLE
Add Generate test case

### PR DESCRIPTION
**Pull request by jasonLaster**
Refs #1004.

### CODE REVIEW

File: test/unit-tests.js [hunk 1]
```diff
@@ -415,9 +396,6 @@ Object.keys(cases).forEach(function (key) {
                 testError(esprima, testCase.case, JSON.parse(testCase.failure));
             } else if (testCase.hasOwnProperty('result')) {
                 testAPI(esprima, testCase.run, JSON.parse(testCase.result));
-            } else {
-                console.error('Incomplete test case:' + testCase.key + '. Generating test result...');
```
Comment (mikesherov) [inline test/unit-tests.js]: will this now silently fail?

Comment (jasonLaster) [inline test/unit-tests.js]: nice catch. just re-added.

**Comment by jasonLaster** [comment]
Note, I made `generate-testcase` an npm script, but it could just as easily be required in and called directly by unit-tests. 

```js
  var generateTestCase require('utils/generate-testcase');

**Comment by jasonLaster** [comment]
After thinking about it, I'll open this PR after the browser tests land. It'll be easier to see what makes sense after `utils/evaluate-test` is available.

**Pull request closed without merging**